dlt/destinations/utils.py

```diff
@@ -57,9 +57,9 @@ def get_resource_for_adapter(data: Any) -> DltResource:
 
 def info_schema_null_to_bool(v: str) -> bool:
     """Converts INFORMATION SCHEMA truth values to Python bool"""
-    if v in ("NO", "0"):
+    if v in ("NO", "0", False):
         return False
-    elif v in ("YES", "1"):
+    elif v in ("YES", "1", True):
         return True
     raise ValueError(v)
```

Comment:
We already created a ticket about this little anomaly.

- https://github.com/crate/crate/issues/17979

Reply:
Hi @rudolfix. Thank you very much for assigning yourself to the PR.

We are currently working on a spin-off to this patch at [dlt-cratedb](https://github.com/crate/dlt-cratedb). After curating the adapter there, we will come back to loop the code into this patch again, subsequently toggling it away from draft mode.